tracking/R3BMDFWrapper.cxx

```diff
@@ -0,0 +1,215 @@
+/******************************************************************************
+ *   Copyright (C) 2019 GSI Helmholtzzentrum für Schwerionenforschung GmbH    *
+ *   Copyright (C) 2019 Members of R3B Collaboration                          *
+ *                                                                            *
+ *             This software is distributed under the terms of the            *
+ *                 GNU General Public Licence (GPL) version 3,                *
+ *                    copied verbatim in the file "LICENSE".                  *
+ *                                                                            *
+ * In applying this license GSI does not waive the privileges and immunities  *
+ * granted to it by virtue of its status as an Intergovernmental Organization *
+ * or submit itself to any jurisdiction.                                      *
+ ******************************************************************************/
+#include"R3BMDFWrapper.h"
+#include <cmath>
+#include<iostream>
+#include<iomanip>
+#include<fstream>
+
+R3BMDFWrapper::R3BMDFWrapper()
+    : mdf_NVariables(0)
+    , pca_NVariables(0)
+    , is_PCA(false)
+{
+}
+
+R3BMDFWrapper::R3BMDFWrapper(const char* mdf_file)
+{
+    InitMDF(mdf_file);
+    is_PCA=false;
+}
+
+R3BMDFWrapper::~R3BMDFWrapper()
+{
+}
+
+//This method will read a special txt file with MDF parameters, don't change
+void R3BMDFWrapper::InitMDF(const char* mdf_file)
+{
+    int i;
+    std::ifstream file(mdf_file);
```

Comment:
Is there any reason to use an input file when they can be loaded as parameters using the standard parameters management in R3BRoot/FAIRRoot? The use of ifstream does not allow control by the RunTimeDB and the files are not included in the parameters repositories... If there is no good reason for that, I would prefer to create a parameter container for this information.

Reply:
Would it be possible to accept the current functionality temporarily? I will prepare the appropriate parameter container later when the rest of the code (tracking task) is developed. Due to some technical reasons (the way how this input file is currently generated, tested, readout etc.) I would prefer to preserve this part of the code.  For sure there will be several other changes in the future and I will try to make everything using standard parameter containers.  At the moment the MDFWrapper class  is necessary for the further code development and several people are waiting for this to be added to the dev branch...

Reply:
ok, no problem